### ISSUE TITLE
GCE instances not able to reach out to internet

### ISSUE THREAD

**Issue by hacktobeer** [forseti-security/osdfir-infrastructure#24]
When we removed the public IP addresses from the turbinia docker worker/server instances they are not able to connect to the internet anymore (NAT??).

**Comment by hacktobeer** [comment]
Enabling Cloud NAT router functionality on the network the VMs are connected to solves this issue.

```
$ gcloud compute create nat-router --network=default --region=us-central1
$ gcloud compute routers nats create nat-config --router-region=us-central1 --router=nat-router --nat-all-subnet-ip-ranges --auto-allocate-nat-external-ips
```

https://cloud.google.com/sdk/gcloud/reference/compute/routers

**Comment by hacktobeer** [comment]
Fixed with #25
